### ISSUE TITLE
Provide first party bulk download

### ISSUE THREAD

**Issue by cthoyt** [bio-tools/biotoolsRegistry#601]
I would like to download the entire database all at once. Having a paginated API isn't ideal for this. It would be great if you had a bulk download link.

Perhaps this could be periodically dumped to `zenodo`

**Comment by cthoyt** [comment]
In the mean time, I wrote a library that extracts everything from the API (https://github.com/cthoyt/biotools-client, takes about 2 hours) and uploaded the results to Zenodo under the CC BY 4.0 license (https://zenodo.org/records/13869530)

**Comment by redmitry** [comment]
I did similar task for tools monitoring...
https://gitlab.bsc.es/inb/elixir/openebench/tools-monitoring/-/blob/main/tools-monitoring-import/biotools-import.py
It converts biotools to bioschemas, so removing all the conversion part and saving to a disk instead of pushing to the server...
It may need some adoption :-(

Cheers,

D.

P.S. importing all biotools should take 10 minutes... 2 hours sounds a lot.

**Comment by veitveit** [comment]
This is a very good point and we are well aware of it. 

The future synchronization with the [ecosystem](https://github.com/research-software-ecosystem) might partly solve this.

I assume a zipped json would be preferred? 

We are backing up the content in as sql files which could be an alternative (after removal of user information).

**Comment by cthoyt** [comment]
Yes, a zipped or otherwise compressed JSON would get the job done! Make sure this archive is on an external system outside of ELIXIR infrastructure, so it will still exist after ELIXIR ends (future planning ;))

**Comment by redmitry** [comment]
Hi All,
I made a simplest bio.tools importer I could. On my notebook it takes 15 min to load all data.
I also noticed, that bio.tools doesn't compress the data sent via the API.
You may save some time and a lot of traffic enabling gzip on the server...

Best,

Dmitry
[backup.json](https://github.com/user-attachments/files/17260553/backup.json)

P.S. I put Apache 2.0 license coz without a license usage of any piece of code may be troublesome....
